product_search_multi_value/data/search_field_data.xml

```diff
@@ -0,0 +1,6 @@
+<odoo noupdate="1">
+    <record model="ir.config_parameter" id="search_fields">
+        <field name="key">product_search_multi_value.search_fields</field>
+        <field name="value">['default_code', 'barcode']</field>
```

Comment:
@Cedric-Pigeon barcode is a wrong example as is a related on product_variant_ids.barcode.

In v10, some product.product are active=False (with product.template active = True), so the generated query for 'not ilike' will exclude products (even templates) for which variants are deactivated.

As I don't see any mean to do that, I will do the tests on another field, set in README the limitation.